static/app/components/searchQueryBuilder/tokens/filterKeyListBox/utils.tsx

```diff
@@ -59,7 +60,7 @@ export function getKeyLabel(
     return `${tag.key}()`;
   }
 
-  return tag.key;
+  return prettifyTagKey(tag.key);
```

Comment:
Can we locate this `prettifyTagKey` function somewhere outside of discover since it's being used in a common component?